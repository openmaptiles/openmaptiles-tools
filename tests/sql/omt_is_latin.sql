SELECT omt_is_latin('') as empty_str;
SELECT omt_is_latin(' ') as space;
SELECT omt_is_latin('abc_xyz') as abc_xyz;
SELECT omt_is_latin('ABC_XYZ') as ABC_XYZ;
SELECT omt_is_latin('0123456789') as digits;

-- Extended Latin from https://en.wikipedia.org/wiki/Latin_Extended_Additional#Compact_table
SELECT omt_is_latin('ḀḁḂḃḄḅḆḇḈḉḊḋḌḍḎḏḐḑḒḓḔḕḖḗḘḙḚḛḜḝḞḟḠḡḢḣḤḥḦḧḨḩḪḫḬḭḮḯḰḱḲḳḴḵḶḷḸḹḺḻḼḽḾḿṀṁṂṃṄṅṆṇṈṉṊṋṌṍṎṏṐṑṒṓṔṕṖṗṘṙṚṛṜṝṞṟṠṡṢṣṤṥṦṧṨṩṪṫṬṭṮṯṰṱṲṳṴṵṶṷṸṹṺṻṼṽṾṿẀẁẂẃẄẅẆẇẈẉẊẋẌẍẎẏẐẑẒẓẔẕẖẗẘẙẚẛẜẝẞẟẠạẢảẤấẦầẨẩẪẫẬậẮắẰằẲẳẴẵẶặẸẹẺẻẼẽẾếỀềỂểỄễỆệỈỉỊịỌọỎỏỐốỒồỔổỖỗỘộỚớỜờỞởỠỡỢợỤụỦủỨứỪừỬửỮữỰựỲỳỴỵỶỷỸỹỺỻỼỽỾỿ') as ext_latin;

-- Russian
SELECT omt_is_latin('привет') as russian_lc;
SELECT omt_is_latin('ПРИВЕТ') as russian_uc;
