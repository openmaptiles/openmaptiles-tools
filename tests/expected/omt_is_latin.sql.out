 empty_str 
-----------
 t
(1 row)

 space 
-------
 t
(1 row)

 abc_xyz 
---------
 t
(1 row)

 abc_xyz 
---------
 t
(1 row)

 digits 
--------
 t
(1 row)

 ext_latin 
-----------
 t
(1 row)

 russian_lc 
------------
 f
(1 row)

 russian_uc 
------------
 f
(1 row)

